include/grpc/grpc_crl_provider.h

```diff
@@ -38,32 +38,26 @@ class Crl {
 
  protected:
   Crl() = default;
-
- private:
-  std::string raw_crl_;
 };
 
-// Opaque representation of a Certificate
-class Cert {
+// Information about a certificate to be used to fetch its associated CRL.
+class CertificateInfo {
```

Comment:
I think this should be an abstract interface.  We can have a concrete impl internally, but those details don't need to be exposed via the API.

Reply:
Will make that change

Reply:
PTAL

Reply:
PTAL